### DISCUSSION TITLE
创建视图中存在反斜杠导致  could not match input

### DISCUSSION THREAD

**Discussion by 13671653088** [apache/doris#6987] [A - General / Q&A]
Doris版本 0.14.12.4 
想了解下，有没有什么办法处理带反斜杠的字符串
分列  select split_part("2019年7\8号", "\\", 1);
替换  select regexp_replace("2019年7\8号",'\\','#');
寻找位置  select find_in_set ( '\\','2\019年78号')

**Comment by EmmyMiao87** [comment]
Try using escape characters?

**Reply by 13671653088** [reply]
I tried but failed, so how to use the escape characters?
select split_part("2019年7\8号" escape '#', "\\", 1);
select split_part("2019年7\8号" , "\\", 1) escape '#';

**Comment by 13671653088** [comment]
最初的问题描述存在一些问题，修改一下
 `select split_part("2019年7\\8号", "\\", 1);`  -- 可以直接执行
`CREATE view test.vw_test as  select split_part("2019年7\\8号", "\\", 1); ` --执行报错 could not match input 无法创建view

解决方案： `create view test.vw_test as  select split_part("2019年7\\\\8号", "\\\\", 1);` 创建view时需要给到双倍的反斜杠
